pysc2/lib/renderer_human.py

```diff
@@ -676,6 +677,25 @@ def draw_overlay(self, surf):
     surf.surf.blit(text, (3, 3))
 
   @sw.decorate
+  def draw_shield_energy_status(self, surf):
+    """Draw the energy and shield status under the unit."""
+    for u, p in self._visible_units():
+      if self._camera.intersects_circle(p, u.radius):
+        if u.alliance != 3:
+          if u.energy or u.shield:
+            text = self.font_small.render(
+                "En: %.0f, Sh: %.0f" % (u.energy, u.shield),
+                True,
+                colors.PLAYER_RELATIVE_PALETTE[u.alliance])
+            rect = text.get_rect()
+
+            x, y = surf.world_to_surf.fwd_pt( p )
```

Comment:
Please follow the google style guide. In this case it means no spaces around arguments.